src/coreComponents/physicsSolvers/fluidFlow/ProppantTransport.cpp

```diff
@@ -101,18 +101,25 @@ void ProppantTransport::registerDataOnMesh( Group & meshBodies )
       subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::proppantConcentrationString() ).
         setDefaultValue( 0.0 ).
         setPlotLevel( PlotLevel::LEVEL_0 );
+
       subRegion.registerWrapper< array1d< real64 > >( viewKeyStruct::deltaProppantConcentrationString() ).
         setDefaultValue( 0.0 );
+
       subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::componentConcentrationString() ).
         setDefaultValue( 0.0 ).
         setPlotLevel( PlotLevel::LEVEL_0 );
+
       subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::deltaComponentConcentrationString() ).
         setDefaultValue( 0.0 );
+
       subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::cellBasedFluxString() ).
-        setDefaultValue( 0.0 ).
         reference().resizeDimension< 1 >( 3 );
+      subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::cellBasedFluxString() ).
+        setDefaultValue( 0.0 );
```

Comment:
Actually, instead of registering two times the same product (there is a protection that mainly returns the version that was already registered), could you just use `getWrapper` and `setDefaultValue` on it?
Or even simpler `auto & w = subRegion.registerWrapper< array2d< real64 > >( viewKeyStruct::cellBasedFluxString() );` and do your stuff directly on `w`?

Registering the same product two times, while under control for the moment, imho, *looks wrong/weird* and we should find another way that people without context will understand for sure.